zebra/zebra_nhg.c

```diff
@@ -3357,16 +3357,22 @@ bool zebra_nhg_proto_nexthops_only(void)
 }
 
 /* Add NHE from upper level proto */
-struct nhg_hash_entry *zebra_nhg_proto_add(uint32_t id, int type,
-					   uint16_t instance, uint32_t session,
+struct nhg_hash_entry *zebra_nhg_proto_add(struct nhg_hash_entry *nhe,
```

Comment:
This commit does not even compile by itself.  NACK

Reply:
fixed.